skfda/exploratory/stats/_stats.py

```diff
@@ -99,6 +100,63 @@ def cov(
     return X.cov(ddof=ddof)
 
 
+@functools.singledispatch
+def std(X: F, ddof: int = 1) -> F:
```

Comment:
Yes, I think we did. `ddof=1` is the default in `numpy.cov`. However, I can change `var` and `std` so that they have the same defult `ddof` values as `numpy.var` and `numpy.std`: both `ddof=0`.

Reply:
Yes, I think we did. `ddof=1` is the default in `numpy.cov`. However, I can change `var` and `std` so that they have the same default `ddof` values as `numpy.var` and `numpy.std`: both `ddof=0`.

Reply:
I thought that we decided the opposite (`ddof = 0`). Please bring this up in the next meeting for clarification. In the future please add the decisions and rationale as comments in the original issue or PR, for future reference.

Reply:
Understood.